Code/Framework/AzCore/AzCore/DOM/DomUtils.h

```diff
@@ -65,29 +65,70 @@ namespace AZ::Dom::Utils
 
     Value DeepCopy(const Value& value, bool copyStrings = true);
 
+    template <typename T>
+    struct is_dom_value
+    {
+        static constexpr bool value = AZStd::is_same_v<AZStd::decay_t<T>, Dom::Value>;
+    };
```

Comment:
You couud just make this a type_alias to AZStd::bool_constant

```suggestion
    template <typename T>
    using is_dom_value = AZStd::bool_constant<AZStd::is_same_v<AZStd::remove_cvref_t<T>, Dom::Value>;
```

Reply:
Done, slightly more concise never hurts.